prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -226,7 +227,7 @@ class GenerateDocs : CliktCommand(
         }
         return Schema(
             fromSchema.baseName,
-            "covid-19",
+            Topic.COVID_19,
```

Comment:
This is not your bug @jimmyfagan but looking at this, the topic shouldn't be hardcoded here when generating schema. We should pull the topic from the schema we're generating documentation for. If we are creating a new schema to write documentation for (not even sure why we are building new) then we should copy the topic from schema. Can you fix this as part of your work here please?

Reply:
Ah good call, fixed now, thanks for catching that!